core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.jelly

```diff
@@ -31,44 +31,50 @@ THE SOFTWARE.
-      <thead>
-        <tr>
-          <th initialSortDir="down">${%Type}</th>
-          <th>${%Name}</th>
-          <th>${%Version}</th>
-          <th>${%Error}</th>
-        </tr>
-      </thead>
-      <tbody>
-        <j:forEach var="item" items="${it.data.entrySet()}">
-          <j:set var="obj" value="${item.key}"/>
-          <j:choose>
-            <j:when test="${item.value!=''}">
-              <tr>
-                <td>${obj.class.name}</td>
-                <!-- fullName is first to avoid calling User.get(String) for User object -->
-                <td>${obj.fullName?:obj.fullDisplayName?:obj.displayName?:obj.name}</td>
-                <td>
-                  <j:choose>
-                    <j:when test="${item.value.isOld(150)}">
-                      <b title="${%Very old version}">${item.value}</b>
-                    </j:when>
-                    <j:otherwise>
-                      ${item.value}
-                    </j:otherwise>
-                  </j:choose>
-                </td>
-                <td style="white-space:normal">${item.value.extra}</td>
-              </tr>
-            </j:when>
-            <j:otherwise>
-              <j:if test="${item.value.extra!=null}"><j:set var="hasExtra" value="${true}"/></j:if>
-            </j:otherwise>
-          </j:choose>
-        </j:forEach>
-      </tbody>
-    </table>
+    <j:if test="${it.data.entrySet().size() > 0}">
+      <table class="jenkins-table sortable">
+        <thead>
+          <tr>
+            <th initialSortDir="down">${%Type}</th>
+            <th>${%Name}</th>
+            <th>${%Version}</th>
+            <th>${%Error}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <j:forEach var="item" items="${it.data.entrySet()}">
+            <j:set var="obj" value="${item.key}"/>
+            <j:choose>
+              <j:when test="${item.value!=''}">
+                <tr>
+                  <td>${obj.class.name}</td>
+                  <!-- fullName is first to avoid calling User.get(String) for User object -->
+                  <td>${obj.fullName?:obj.fullDisplayName?:obj.displayName?:obj.name}</td>
+                  <td>
+                    <j:choose>
+                      <j:when test="${item.value.isOld(150)}">
+                        <b title="${%Very old version}">${item.value}</b>
+                      </j:when>
+                      <j:otherwise>
+                        ${item.value}
+                      </j:otherwise>
+                    </j:choose>
+                  </td>
+                  <td style="white-space:normal">${item.value.extra}</td>
+                </tr>
+              </j:when>
+              <j:otherwise>
+                <j:if test="${item.value.extra!=null}"><j:set var="hasExtra" value="${true}"/></j:if>
+              </j:otherwise>
+            </j:choose>
+          </j:forEach>
+        </tbody>
+      </table>
+    </j:if>
+      <j:if test="${it.data.entrySet().size() == 0}">
```

Comment:
instead of adding 2 `j:if`'s this should use a `j:choose` see example below, other that the `j:otherwise` shows the `l:notice`